test/metabase/api/permissions_test.clj

```diff
@@ -59,7 +59,7 @@
       (is (= (mt/user-http-request :crowberto :get 200 "permissions/group" :offset "1" :limit 50)
              (mt/user-http-request :crowberto :get 200 "permissions/group" :offset "1"))))
     (testing "Limit and offset pagination works for permissions list"
-      (is (= [{:id 1, :name "All Users", :member_count 3}]
+      (is (partial= [{:id 1, :name "All Users"}]
```

Comment:
This was failing locally since I had an extra user in my DB. Changed it to a partial assertion since `member_count` isn't relevant for this test.